srcpkgs/cdk/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'cdk'
+pkgname=cdk
+version=5.0
+revision=1
+_date=20190303
+wrksrc="cdk-${version}-${_date}"
+build_style=gnu-configure
+configure_args="--with-ncurses=${XBPS_CROSS_BASE}/usr/lib"
+makedepends="ncurses-devel"
+short_desc="Curses Development Kit - widget library, enhanced version"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-2.0-only, BSD-4-Clause"
+homepage="https://invisible-island.net/cdk/"
+distfiles="https://fossies.org/linux/misc/cdk-${version}-${_date}.tgz"
+checksum=adad7870988d44b95df57722ab8dffc587d035183eb6b12a9500ebed4d8dba25
+
+post_install() {
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	depends="${sourcepkg}-${version}_${revision} ncurses-devel"
+	short_desc+=" -development files"
+	pkg_install() {
+		vmove usr/bin/cdk5-config
+		vmove usr/include
+		vmove "usr/lib/*.a"
```

Comment:
no shared library this time? sounds weird

Reply:
Yep if we do static, we don't get shared. If I enable shared, no static. Hmm, may have idea, will have to dive into the build script.

Reply:
no way buddy. two of them or nothing

Reply:
I have manually created the static library and installed.